In [3]:
import pandas as pd

In [4]:
assam_exp_2018_19 = pd.read_csv("/home/ubuntu/Downloads/assam_processed_data (1) (1).csv") 

Number of rows :- **27063**
Number of columns :- **20**

In [5]:
assam_exp_2018_19.shape

(27063, 20)

In [7]:
assam_exp_2018_19.describe()

#  ACTUALS 2016-17  BUDGET 2017-18  REVISED 2017-18  \
count  27063.000000     27063.000000    27063.000000     27063.000000   
mean   13532.000000       153.950093      317.493007       367.514263   
std     7812.559504      3718.536445     5119.113574      5464.405369   
min        1.000000       -33.700000       -0.010000        -0.010000   
25%     6766.500000         0.000000        0.100000         0.150000   
50%    13532.000000         0.000000        1.230000         1.500000   
75%    20297.500000         0.000000       14.000000        16.870000   
max    27063.000000    405739.690000   568750.000000    568750.000000   

       BUDGET 2018-19  
count    27063.000000  
mean       333.554734  
std       5100.622950  
min          0.000000  
25%          0.160000  
50%          2.000000  
75%         17.560000  
max     474011.040000

* **Budget Estimates** : Estimates of government spending on various sectors during the year
* **Revised Estimates** : Is the difference between the Previous Budget Estimates and the actual expenditure, which is usually presented in the following Budget.
* **Actuals** : These represent actual figures that were being spent in that year

Total expenditure figures each year are(figures are in lakhs):-
* ACTUALS 2016-17 -> **4166351.3699999996**
* BUDGET 2017-18 -> **8592313.248850001**
* REVISED 2017-18 -> **9946038.486489998**
* BUDGET 2018-19 -> **9026991.767211499**

There is 6-tier classification of budgets to record and classify the receipts and expenditure relating to functions of the government. The tiers are as follows:
* **Major Heads**
* **Sub-major Heads**
* **Minor Heads**
* **Sub-minor Heads**
* **Detailed Heads**
* **Voucher Head**
* **Object Heads**

And the hierarchy continues as


**Major Head -> Sub-Major Head -> Minor Head -> Sub-Minor Head -> Detailed Head -> Object Head -> Voucher Head**

## **Grant Number**
There are total **81** different grants with grant numbers from **1-78** and **3** alpha initials
* max expenditure budget 2018-19 -> 71 - Education (Elementary,Scy.Etc.) **1361921.4043900005**
* min expenditure budget 2018-19 -> 24 - Aid Materials **1** 

In [7]:
grant_wise = assam_exp_2018_19.groupby('GRANT NUMBER').sum()

### **Top five grants**

In [20]:
grant_wise.sort_values(['BUDGET 2018-19'], ascending= 0).head()

#  ACTUALS 2016-17  \
GRANT NUMBER                                                        
71 - Education (Elementary,Scy.Etc.)    12895535        923563.14   
PD - Public Debt And Servicing Of Debt    761891        384242.83   
23 - Pension                              327709        655413.19   
14 - Police                             13070213        282785.52   
29 - Medical & Public Health            24419212        247782.33   

                                        BUDGET 2017-18  REVISED 2017-18  \
GRANT NUMBER                                                              
71 - Education (Elementary,Scy.Etc.)      1.093981e+06     1.183154e+06   
PD - Public Debt And Servicing Of Debt    8.660136e+05     8.660136e+05   
23 - Pension                              9.659683e+05     9.659683e+05   
14 - Police                               4.631736e+05     5.138987e+05   
29 - Medical & Public Health              4.634636e+05     5.452227e+05   

                                        BUDGET 2018-19  
GRANT NUMBER                                            
71 - Education (Elementary,Scy.Etc.)      1.361921e+06  
PD - Public Debt And Servicing Of Debt    8.252711e+05  
23 - Pension                              8.005794e+05  
14 - Police                               5.106355e+05  
29 - Medical & Public Health              5.082168e+05

### **Bottom five grants**

In [23]:
grant_wise.sort_values(['BUDGET 2018-19'], ascending= 0).tail()

#  ACTUALS 2016-17  BUDGET 2017-18  \
GRANT NUMBER                                                         
10 - Other Fiscal Service  140266           180.35        269.3336   
32 - Housing Schemes       172459            83.35        501.6700   
28 - State Archives        296895           126.22        189.8000   
47 - Trade advisor         417287            71.65        109.7900   
24 - Aid Materials           8876             0.00          1.0000   

                           REVISED 2017-18  BUDGET 2018-19  
GRANT NUMBER                                                
10 - Other Fiscal Service         271.8336         299.948  
32 - Housing Schemes             1851.6700         279.000  
28 - State Archives               189.8000         234.490  
47 - Trade advisor                115.9400         148.770  
24 - Aid Materials                  1.0000           1.000

## **Major Head**

There are total **126** different major heads
* max expenditure budget 2018-19 -> General Education **1.690370e+06**
* min expenditure budget 2018-19 -> 24 - Capital Outlay on Agricultural Research and Education **0** 

In [25]:
major_head_wise = assam_exp_2018_19.groupby('Major Head').sum()

### **Top five major heads**

In [29]:
major_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).head()

#  ACTUALS 2016-17  \
Major Head                                                         
General Education                      28180808       1203135.23   
Pension and Other Retirement Benefits    331503        656464.21   
Police                                 12829589        282727.62   
Medical and Public Health              40547073        242317.53   
Interest Payments                        429385        289602.28   

                                       BUDGET 2017-18  REVISED 2017-18  \
Major Head                                                               
General Education                        1.438585e+06     1.557220e+06   
Pension and Other Retirement Benefits    9.682383e+05     9.682383e+05   
Police                                   4.449785e+05     4.956653e+05   
Medical and Public Health                4.131204e+05     4.823343e+05   
Interest Payments                        3.746748e+05     3.746748e+05   

                                       BUDGET 2018-19  
Major Head                                             
General Education                        1.690370e+06  
Pension and Other Retirement Benefits    8.028494e+05  
Police                                   4.917688e+05  
Medical and Public Health                4.335569e+05  
Interest Payments                        4.193346e+05

### **Bottom five major heads**

In [30]:
major_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).tail()

#  ACTUALS 2016-17  \
Major Head                                                                    
Aid Materials & Equipments                            8876             0.00   
Secretariat Social Services                         357584          3413.72   
Loans for Housing                                    36957             0.00   
Appropriation to the Contingency Fund                41005             0.00   
Capital Outlay on Agricultural Research and Edu...  182510             0.00   

                                                    BUDGET 2017-18  \
Major Head                                                           
Aid Materials & Equipments                                 1.00000   
Secretariat Social Services                             3955.02074   
Loans for Housing                                        252.00000   
Appropriation to the Contingency Fund                   5000.00000   
Capital Outlay on Agricultural Research and Edu...      6200.00000   

                                                    REVISED 2017-18  \
Major Head                                                            
Aid Materials & Equipments                                  1.00000   
Secretariat Social Services                              3955.02074   
Loans for Housing                                         252.00000   
Appropriation to the Contingency Fund                   10000.00000   
Capital Outlay on Agricultural Research and Edu...       6300.00000   

                                                    BUDGET 2018-19  
Major Head                                                          
Aid Materials & Equipments                                    1.00  
Secretariat Social Services                                   0.84  
Loans for Housing                                             0.00  
Appropriation to the Contingency Fund                         0.00  
Capital Outlay on Agricultural Research and Edu...            0.00

## **Sub-Major Head**

There are total **101** different sub-major heads
* max expenditure budget 2018-19 -> Elementary Education **898016.77382**
* min expenditure budget 2018-19 -> Storage and Warehousing **0** 

In [32]:
sub_major_head_wise = assam_exp_2018_19.groupby('Sub-Major Head').sum()

### **Top five Sub-Major Heads**

In [34]:
sub_major_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).head()

#  ACTUALS 2016-17  BUDGET 2017-18  \
Sub-Major Head                                                    
Elementary Education   7644056        686706.18    797364.43168   
Civil                   331503        656464.21    968238.28000   
General               25435839        173523.91    376383.22210   
Secondary Education    6830793        327698.11    410995.08200   
State Highways         2081633        126396.94    273125.44500   

                      REVISED 2017-18  BUDGET 2018-19  
Sub-Major Head                                         
Elementary Education     848540.48168    898016.77382  
Civil                    968238.28000    802849.38000  
General                  649187.51210    562325.65386  
Secondary Education      461324.75200    529504.97500  
State Highways           310674.07500    320561.75200

### **Bottom five Sub-major heads**

In [35]:
sub_major_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).tail()

#  ACTUALS 2016-17  BUDGET 2017-18  \
Sub-Major Head                                                     
Non Plan Loans             1000              0.0             0.0   
Consumer Industries       15142            250.0           250.0   
Crop Husbandary          165070              0.0          6100.0   
Animal Husbandary         17440              0.0           100.0   
Storage and Warehousing   24702              0.0           130.0   

                         REVISED 2017-18  BUDGET 2018-19  
Sub-Major Head                                            
Non Plan Loans                      0.00            0.10  
Consumer Industries              1436.68            0.01  
Crop Husbandary                  6200.00            0.00  
Animal Husbandary                 100.00            0.00  
Storage and Warehousing           130.00            0.00

## **Minor Head**

There are total **401** different minor heads
* max expenditure budget 2018-19 -> Other Expenditure **1.939502e+06**
* min expenditure budget 2018-19 -> Deduct amount met from SDRF & transfer to 8121 general and Other Reserve Funds **0** 

In [37]:
minor_head_wise = assam_exp_2018_19.groupby('Minor Head').sum()

### **Top five Minor Heads**

In [40]:
minor_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).head()

#  ACTUALS 2016-17  \
Minor Head                                                                  
Other Expenditure                               33558086        704919.04   
Government Primary Schools                       1883340        463455.58   
Direction and Administration                    76088003        317261.80   
Superannuation and Retirement Allowances           88590        411022.70   
Assistance to Non-Government Secondary Schools   1582925        270794.13   

                                                BUDGET 2017-18  \
Minor Head                                                       
Other Expenditure                                 1.756176e+06   
Government Primary Schools                        5.322728e+05   
Direction and Administration                      4.488919e+05   
Superannuation and Retirement Allowances          5.844410e+05   
Assistance to Non-Government Secondary Schools    3.414697e+05   

                                                REVISED 2017-18  \
Minor Head                                                        
Other Expenditure                                  2.311148e+06   
Government Primary Schools                         5.618878e+05   
Direction and Administration                       5.575137e+05   
Superannuation and Retirement Allowances           5.844410e+05   
Assistance to Non-Government Secondary Schools     3.905687e+05   

                                                BUDGET 2018-19  
Minor Head                                                      
Other Expenditure                                 1.939502e+06  
Government Primary Schools                        5.902927e+05  
Direction and Administration                      5.631495e+05  
Superannuation and Retirement Allowances          4.866658e+05  
Assistance to Non-Government Secondary Schools    4.547502e+05

### **Bottom five Minor heads**

In [41]:
minor_head_wise .sort_values(['BUDGET 2018-19'], ascending= 0).tail()

#  ACTUALS 2016-17  \
Minor Head                                                                   
NBA Sanitation Services                             24351              0.0   
Assistant to Trading Institutions                   20012              0.0   
Welfare of Aged Infirm & Destitute (SW)             14523              0.0   
Loans to Housing Boards                             36957              0.0   
Deduct amount met from SDRF & transfer to 8121 ...  14943              0.0   

                                                    BUDGET 2017-18  \
Minor Head                                                           
NBA Sanitation Services                                   33405.67   
Assistant to Trading Institutions                             0.00   
Welfare of Aged Infirm & Destitute (SW)                       0.01   
Loans to Housing Boards                                     252.00   
Deduct amount met from SDRF & transfer to 8121 ...           -0.01   

                                                    REVISED 2017-18  \
Minor Head                                                            
NBA Sanitation Services                                    33405.67   
Assistant to Trading Institutions                            140.00   
Welfare of Aged Infirm & Destitute (SW)                        0.01   
Loans to Housing Boards                                      252.00   
Deduct amount met from SDRF & transfer to 8121 ...            -0.01   

                                                    BUDGET 2018-19  
Minor Head                                                          
NBA Sanitation Services                                        0.0  
Assistant to Trading Institutions                              0.0  
Welfare of Aged Infirm & Destitute (SW)                        0.0  
Loans to Housing Boards                                        0.0  
Deduct amount met from SDRF & transfer to 8121 ...             0.0

## **Sub-Minor Head**

There are total **2511** different Sub-minor heads
* max expenditure budget 2018-19 -> Government teachers serving in Non-Govt. Schools **453281.273**
* min expenditure budget 2018-19 -> state specific fund for organization of outreach programme **0** 

In [42]:
sub_minor_head_wise = assam_exp_2018_19.groupby('Sub-Minor Head').sum()

### **Top five Sub-Minor Heads**

In [44]:
sub_minor_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).head()

#  ACTUALS 2016-17  \
Sub-Minor Head                                                               
Government teachers serving in Non-Govt. Schools  1230990        270701.28   
Government Primary School                          556279        256761.13   
National Health Mission (NHM)                      411367        104687.50   
Government Middle School                          1119277        206624.14   
Special W & M Advance                                 133             0.00   

                                                  BUDGET 2017-18  \
Sub-Minor Head                                                     
Government teachers serving in Non-Govt. Schools    340173.34000   
Government Primary School                           316500.67643   
National Health Mission (NHM)                       190337.00000   
Government Middle School                            215609.50912   
Special W & M Advance                               180000.00000   

                                                  REVISED 2017-18  \
Sub-Minor Head                                                      
Government teachers serving in Non-Govt. Schools     389149.20000   
Government Primary School                            337790.58643   
National Health Mission (NHM)                        223902.99000   
Government Middle School                             223934.68912   
Special W & M Advance                                180000.00000   

                                                  BUDGET 2018-19  
Sub-Minor Head                                                    
Government teachers serving in Non-Govt. Schools      453281.273  
Government Primary School                             378150.890  
National Health Mission (NHM)                         224171.620  
Government Middle School                              211977.640  
Special W & M Advance                                 180000.000

### **Bottom five Sub-Minor heads**

In [45]:
sub_minor_head_wise .sort_values(['BUDGET 2018-19'], ascending= 0).tail()

#  ACTUALS 2016-17  \
Sub-Minor Head                                                               
Publication of Books                                14040              0.0   
Public Service Information Systems                  21495              0.0   
Construction Of Chowkidar Shed at CFC, Water Hy...  18724              0.0   
Construction Of Staff Quarter & Catering At TRH...  13784              0.0   
state specific fund for organization of outreac...  16825              0.0   

                                                    BUDGET 2017-18  \
Sub-Minor Head                                                       
Publication of Books                                          5.00   
Public Service Information Systems                            0.01   
Construction Of Chowkidar Shed at CFC, Water Hy...           20.00   
Construction Of Staff Quarter & Catering At TRH...           50.00   
state specific fund for organization of outreac...           50.00   

                                                    REVISED 2017-18  \
Sub-Minor Head                                                        
Publication of Books                                           5.00   
Public Service Information Systems                             0.01   
Construction Of Chowkidar Shed at CFC, Water Hy...            20.00   
Construction Of Staff Quarter & Catering At TRH...            50.00   
state specific fund for organization of outreac...            50.00   

                                                    BUDGET 2018-19  
Sub-Minor Head                                                      
Publication of Books                                           0.0  
Public Service Information Systems                             0.0  
Construction Of Chowkidar Shed at CFC, Water Hy...             0.0  
Construction Of Staff Quarter & Catering At TRH...             0.0  
state specific fund for organization of outreac...             0.0

## **Detailed Head**

There are total **1867** different detailed heads
* max expenditure budget 2018-19 -> Central Share **295094.590**
* min expenditure budget 2018-19 -> upgradation of approach road to mohanbari airport, Dibrugarh including widening of roadside drain **0** 

In [47]:
detailed_head_wise = assam_exp_2018_19.groupby('Detailed Head').sum()

### **Top five Detailed Heads**

In [49]:
detailed_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).head()

#  ACTUALS 2016-17  BUDGET 2017-18  \
Detailed Head                                                               
Central Share                     329547        117537.93       220745.75   
Null                             2288431         15349.94        45660.80   
Assistance to Gaon panchayat       55587             0.00       174536.00   
Contribution of Pension of ASEB    19510             0.00        20000.00   
PMGSY Works (Block Grant)          19993            43.83        44220.00   

                                 REVISED 2017-18  BUDGET 2018-19  
Detailed Head                                                     
Central Share                        290675.2000      295094.590  
Null                                  54506.5025      198744.904  
Assistance to Gaon panchayat         174536.0000      121140.000  
Contribution of Pension of ASEB       40000.0000      116387.000  
PMGSY Works (Block Grant)             58204.0000       84453.000

### **Bottom five Detailed heads**

In [51]:
detailed_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).tail()

#  ACTUALS 2016-17  \
Detailed Head                                                                
Infrastructure Development of LKRB Music College     9721             0.00   
Infrastructure Development of Government Arts &...   9720             0.00   
Imrov. of Bordoloni to Khalihamari (SBG) Road i...  19971           138.43   
Improving riding quality of roads structure inc...  20001             0.00   
upgradation of approach road to mohanbari airpo...  20000             0.00   

                                                    BUDGET 2017-18  \
Detailed Head                                                        
Infrastructure Development of LKRB Music College            200.00   
Infrastructure Development of Government Arts &...          200.00   
Imrov. of Bordoloni to Khalihamari (SBG) Road i...           97.21   
Improving riding quality of roads structure inc...           42.72   
upgradation of approach road to mohanbari airpo...           21.33   

                                                    REVISED 2017-18  \
Detailed Head                                                         
Infrastructure Development of LKRB Music College             200.00   
Infrastructure Development of Government Arts &...           200.00   
Imrov. of Bordoloni to Khalihamari (SBG) Road i...            97.21   
Improving riding quality of roads structure inc...            42.72   
upgradation of approach road to mohanbari airpo...            21.33   

                                                    BUDGET 2018-19  
Detailed Head                                                       
Infrastructure Development of LKRB Music College               0.0  
Infrastructure Development of Government Arts &...             0.0  
Imrov. of Bordoloni to Khalihamari (SBG) Road i...             0.0  
Improving riding quality of roads structure inc...             0.0  
upgradation of approach road to mohanbari airpo...             0.0

## **Object Head**

There are total **29** different Object heads
* max expenditure budget 2018-19 -> Government teachers serving in Non-Govt. Schools **2.646989e+06**
* min expenditure budget 2018-19 -> Grants-in-Aid/Contribution/Subsidies **0** 

In [60]:
Object_head_wise = assam_exp_2018_19.groupby('Object Head').sum()

### **Top five Object Heads**

In [58]:
Object_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).head()

#  ACTUALS 2016-17  \
Object Head                                                      
Salaries                            149416190       1800250.47   
Grants-in-aid General (Non-Salary)   18023075        584579.78   
Major Works                          16765309        247020.52   
Loans                                 1262844        383898.99   
Pension / Gratuity                     778431        663061.89   

                                    BUDGET 2017-18  REVISED 2017-18  \
Object Head                                                           
Salaries                              2.498331e+06     2.860891e+06   
Grants-in-aid General (Non-Salary)    1.910879e+06     2.381295e+06   
Major Works                           7.883047e+05     8.718326e+05   
Loans                                 8.913068e+05     8.997561e+05   
Pension / Gratuity                    9.768486e+05     9.771318e+05   

                                    BUDGET 2018-19  
Object Head                                         
Salaries                              2.646989e+06  
Grants-in-aid General (Non-Salary)    1.953669e+06  
Major Works                           8.841306e+05  
Loans                                 8.567468e+05  
Pension / Gratuity                    8.158601e+05

### **Bottom five Object heads**

In [59]:
Object_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).tail()

#  ACTUALS 2016-17  \
Object Head                                                                 
Publication                                      2346063          1020.91   
Secret Service Expenditure                         88288          1291.76   
Hospitality Expenses / Sumptuary Allowances etc   829779           278.90   
Writes off  / Losses                              203723             0.00   
Grants-in-Aid/Contribution/Subsidies               15166          1010.22   

                                                 BUDGET 2017-18  \
Object Head                                                       
Publication                                           2318.7910   
Secret Service Expenditure                            1367.8000   
Hospitality Expenses / Sumptuary Allowances etc        340.6585   
Writes off  / Losses                                    73.9600   
Grants-in-Aid/Contribution/Subsidies                    50.0000   

                                                 REVISED 2017-18  \
Object Head                                                        
Publication                                            2349.7910   
Secret Service Expenditure                             1367.8000   
Hospitality Expenses / Sumptuary Allowances etc         442.2985   
Writes off  / Losses                                     73.9600   
Grants-in-Aid/Contribution/Subsidies                     50.0000   

                                                 BUDGET 2018-19  
Object Head                                                      
Publication                                            2075.897  
Secret Service Expenditure                             1491.850  
Hospitality Expenses / Sumptuary Allowances etc         532.042  
Writes off  / Losses                                     13.960  
Grants-in-Aid/Contribution/Subsidies                      0.000

## **Voucher Head**

There are total **161** different Voucher heads
* max expenditure budget 2018-19 -> Others Schools **4.021223e+06**
* min expenditure budget 2018-19 -> Special Police Officer **0** 

In [64]:
Voucher_head_wise = assam_exp_2018_19.groupby('Voucher Head').sum()

### **Top five Voucher Heads**

In [66]:
Voucher_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).head()

#  ACTUALS 2016-17  BUDGET 2017-18  \
Voucher Head                                                     
Others               92237731       1132551.95    3.467864e+06   
Pay                  18932433       1788299.17    8.874673e+05   
Pension                320659        505862.47    7.018059e+05   
Interest Payment       429385        289602.28    3.746748e+05   
Principal Repayment    332183         77119.53    4.703413e+05   

                     REVISED 2017-18  BUDGET 2018-19  
Voucher Head                                          
Others                  4.350240e+06    4.021223e+06  
Pay                     1.055147e+06    2.046766e+06  
Pension                 7.019291e+05    5.902418e+05  
Interest Payment        3.746748e+05    4.193346e+05  
Principal Repayment     4.703413e+05    3.830311e+05

### **Bottom five Voucher heads**

In [67]:
Voucher_head_wise.sort_values(['BUDGET 2018-19'], ascending= 0).tail()

#  ACTUALS 2016-17  \
Voucher Head                                                       
Organisation of Events/ Fair & Functions  38572             0.00   
RISK FUND                                  3818             0.00   
INCENTIVES                                 7638             0.00   
Contribution                              15166          1010.22   
Special Police Officer                     5820             4.97   

                                          BUDGET 2017-18  REVISED 2017-18  \
Voucher Head                                                                
Organisation of Events/ Fair & Functions           22.11            22.11   
RISK FUND                                       10000.00         10000.00   
INCENTIVES                                      35200.00         35200.00   
Contribution                                       50.00            50.00   
Special Police Officer                              1.00             1.00   

                                          BUDGET 2018-19  
Voucher Head                                              
Organisation of Events/ Fair & Functions             0.0  
RISK FUND                                            0.0  
INCENTIVES                                           0.0  
Contribution                                         0.0  
Special Police Officer                               0.0

## **Number of Schemes**

In [69]:
schemes = assam_exp_2018_19.groupby('Scheme').sum()

In [72]:
schemes.sort_values(['BUDGET 2018-19'], ascending= 0)

#  \
Scheme                                                          
Establishmet Expenditure                            300034193   
Centrally Sponsored Scheme                           11018407   
State Own Priority Scheme-General                    30673259   
State Own Priority Scheme-Other Development Scheme    5903935   
State Own Priority Scheme-State Share                 3461527   
Rural Infrastructure Development fund-Loan Share       398557   
Externally Aided Projects                              265908   
Transfer Grants to Urban Local Bodies                  407691   
State Own Priority Scheme-GOI Special Scheme          2013570   
Establishment Expenditure-Central Share               3893855   
Transfer Grants to Autonomous Councils                 240598   
Transfer Grants to State Finance Commission Grants     350147   
Transfer Grants to Finance Commission Grants           147892   
Transfer Grants to Educational Institutions            315980   
Externally Aided Projects-State Share                  234435   
Transfer Grants to Sixth Schedule Areas                428652   
State Own Priority Scheme-SCSP                         822576   
Transfer Grants to Individual Benefeciaries            211626   
Rural Infrastructure Development fund-State Share      340605   
Establishmet Expenditure-State Share                   131909   
State Own Priority Scheme-TSP                          775273   
Establishment Expenditure-Six Schedule Area           3221800   
Transfer Grants to Development Councils                854874   
State Own Priority Scheme-SCSP State Share              41937   
Plan                                                    27310   

                                                    ACTUALS 2016-17  \
Scheme                                                                
Establishmet Expenditure                                 2990839.48   
Centrally Sponsored Scheme                                514270.38   
State Own Priority Scheme-General                         331761.52   
State Own Priority Scheme-Other Development Scheme         12789.06   
State Own Priority Scheme-State Share                      36620.46   
Rural Infrastructure Development fund-Loan Share           74258.51   
Externally Aided Projects                                  55310.91   
Transfer Grants to Urban Local Bodies                        721.67   
State Own Priority Scheme-GOI Special Scheme               18304.01   
Establishment Expenditure-Central Share                    64604.47   
Transfer Grants to Autonomous Councils                     30416.11   
Transfer Grants to State Finance Commission Grants             0.00   
Transfer Grants to Finance Commission Grants                   0.00   
Transfer Grants to Educational Institutions                 8109.96   
Externally Aided Projects-State Share                      14145.03   
Transfer Grants to Sixth Schedule Areas                        0.00   
State Own Priority Scheme-SCSP                              7177.71   
Transfer Grants to Individual Benefeciaries                  198.05   
Rural Infrastructure Development fund-State Share           3075.28   
Establishmet Expenditure-State Share                         775.49   
State Own Priority Scheme-TSP                               1901.49   
Establishment Expenditure-Six Schedule Area                 1037.26   
Transfer Grants to Development Councils                       34.52   
State Own Priority Scheme-SCSP State Share                     0.00   
Plan                                                           0.00   

                                                    BUDGET 2017-18  \
Scheme                                                               
Establishmet Expenditure                              4.649374e+06   
Centrally Sponsored Scheme                            1.543443e+06   
State Own Priority Scheme-General                     5.847649e+05   
State Own Priority Scheme-Other Deve

## **Number of Areas**

In [73]:
areas = assam_exp_2018_19.groupby('Area').sum()

In [74]:
areas.sort_values(['BUDGET 2018-19'], ascending= 0)

#  ACTUALS 2016-17  BUDGET 2017-18  \
Area                                                                      
General Area                 220319465       3882246.16    8.077832e+06   
Bodoland Entrusted            57043006        170534.97    2.184226e+05   
Karbi Anglong Entrusted       41892468         61133.87    1.208489e+05   
North Cachar Entrusted        35214424         30297.45    4.929969e+04   
Bodoland Non-entrusted         1924761          7226.31    7.015690e+04   
Karbi Anglong Non-entrusted    5411340          8292.98    3.715705e+04   
North Cachar Non-entrusted     4411052          6619.63    1.859592e+04   

                             REVISED 2017-18  BUDGET 2018-19  
Area                                                          
General Area                    9.312938e+06    8.477015e+06  
Bodoland Entrusted              2.473677e+05    2.786262e+05  
Karbi Anglong Entrusted         1.415349e+05    1.397418e+05  
North Cachar Entrusted          5.463205e+04    6.767248e+04  
Bodoland Non-entrusted          1.097821e+05    2.897986e+04  
Karbi Anglong Non-entrusted     5.306732e+04    2.313319e+04  
North Cachar Non-entrusted      2.671685e+04    1.182317e+04